__tests__/html/transcript.navigation.visualKeyboardIndicator.html

```diff
@@ -52,36 +52,7 @@
 
         // THEN: It should show a focus indicator around the transcript, and an active indicator around the last activity.
         await host.snapshot();
-
-        // WHEN: SHIFT-TAB key is pressed from the transcript.
-        await host.sendShiftTab();
-
-        // THEN: It should show the keyboard help.
-        await host.snapshot();
-
-        // WHEN: SHIFT-TAB key is pressed from the keyboard help.
-        await host.sendShiftTab();
```

Comment:
I think you should remove only these two lines, everything bellow should work as before. May also need to update snapshots to reflect changes.